packages/preview/tufte-memo/0.1.0/lib.typ

```diff
@@ -0,0 +1,314 @@
+#import "@preview/drafting:0.2.0": *
+
+#let wideblock(content) = block(width:100%+2.5in,content)
+
+// Fonts used in front matter, sidenotes, bibliography, and captions
+#let sans-fonts = (
+    "Gill Sans MT",
+    "TeX Gyre Heros",
+    "Noto Sans"
+  )
+
+// Fonts used for headings and body copy
+#let serif-fonts = (
+  "Lucida Bright",
+  "New CM",
+  "Linux Libertine",
+)
+
+#let template(
+  title: [Paper Title],
+  shorttitle: none,
+  subtitle: none,
+  authors: (
+    (
+      name: "First Last",
+      role: none,
+      organization: none,
+      location: none,
+      email: none
+    ),
+  ),
+  date: datetime.today(),
+  document_number: none,
```

Comment:
To be consistent with the rest of the package and of the Typst ecosystem, this variable should probably use kebab-case and be renamed to `document-number`.